data/raw/keybindings.json

```diff
@@ -1680,13 +1680,6 @@
       { "input_method": "keyboard_code", "key": "KEYPAD_5" }
     ]
   },
-  {
-    "type": "keybinding",
-    "name": "Toggle Map Memory",
-    "category": "DEFAULTMODE",
-    "id": "toggle_map_memory",
-    "bindings": [ { "input_method": "keyboard_char", "key": "{" }, { "input_method": "keyboard_code", "key": "[", "mod": [ "shift" ] } ]
```

Comment:
I think we delete just a single row with `bindings`. Otherwise action would lack description in keybindings menu.

Reply:
you're right

will fix later (or someone else can push an edit to this)